hub/router/routes.go

```diff
@@ -171,7 +171,7 @@ func AddUser(c *gin.Context) {
 		return
 	}
 
-	if user == (models.User{}) {
+	if user.Username == "" || user.Password == "" || (user.Role == "user" && len(user.WorkspaceIDs) == 0) {
```

Comment:
Same as [here](https://github.com/shamanec/GADS/pull/174/files#r2005383127)

Reply:
I'm not sure what should be changed in this code section... This code is part of user creation, so it's expected that username, password, and possible workspaces are valid, right?

Reply:
Ah, you are right, I didn't look properly :D